label_options/caebike-label-options.json

```diff
@@ -0,0 +1,85 @@
+{
+  "MODE": [
+    {"value":"walk", "baseMode":"WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}}, "kgCo2PerKm": 0},
+    {"value":"auto_driver", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
+    {"value":"auto_passenger", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
+    {"value":"e-bike", "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
+    {"value":"bike", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
+    {"value":"taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+    {"value":"bike_scooter_share", "baseMode":"BICYCLING_SCOOTER_SHARE", "met": {"ALL": {"range": [0, -1], "mets": 3.5}}, "kgCo2PerKm": 0.00447},
+    {"value":"bus_train_shuttle", "baseMode":"PUBLIC_TRANSIT", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.179033},
+    {"value":"other_mode", "baseMode":"OTHER_MODE", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0},
```

Comment:
I think that this needs to be "other" for the "other" logic in the multi-label, which allows the user to enter a custom value, to take over.

Reply:
I put in "other mode" because we would like to nudge the user to select one of the pre-defined mode categories if possible. And in cases where the user would still want to select "other mode", we don't want to burden the user by requiring them to specify what the other mode is.

Reply:
@si-agarwal Ah yes, I remember that conversation. But we had also discussed that given that the mode that the user selects shows up in their mode list next time, allowing users to type in the other mode would also allow them to create their own, user-specific mode list. This would allow us to explore the long tail of < 5% mode share modes without imposing a significant burden on the user. The example for that was "motorcycle" from the time Chris was commuting to Berkeley.

Did you decide not to go down that route?

Reply:
I'll discuss this with @dtfitch

Reply:
Hi @shankari, it is my and @dtfitch 's recollection that the group settled on displaying uniform mode lists to all users with the mode options listed in the json file.

Reply:
Ok! You can always change it later if you feel it is too limiting...